.travis.yml

```diff
@@ -86,6 +86,21 @@ script:
   - mvn install -DskipTests=true -B -pl janusgraph-${MODULE} -am
   - mvn verify -pl janusgraph-${MODULE} ${ARGS}
 
+before_deploy:
+  - mvn package -DskipTests -Dgpg.skip=true -Pjanusgraph-release
```

Comment:
I never tried to work through getting GPG signing to work as part of a Traivs CI job. Not sure if it's possible. If not maybe deploying in this way is a non-starter? I do like the general idea of handling release deployment as part of the CI process though.